core/src/components/datetime/datetime.tsx

```diff
@@ -2011,7 +2042,11 @@ export class Datetime implements ComponentInterface {
 
           <div class="calendar-next-prev">
             <ion-buttons>
-              <ion-button aria-label="Previous month" disabled={prevMonthDisabled} onClick={() => this.prevMonth()}>
+              <ion-button
+                aria-label="Previous month"
+                disabled={prevMonthDisabled || disabled}
```

Comment:
We should be updating the `prevMonthDisabled` constant with the additional check for `disabled`.